components/director/run.sh

```diff
@@ -75,9 +77,9 @@ set -e
 echo -e "${GREEN}Populate DB${NC}"
 
 cat ${ROOT_PATH}/../schema-migrator/migrations/director/*.up.sql | \
-    docker exec -i ${POSTGRES_CONTAINER} psql -U "$DB_USER" -h "$DB_HOST" -p "$DB_PORT" -d "$DB_NAME"
+    docker exec -i ${POSTGRES_CONTAINER} bash -c 'psql -U "$POSTGRES_USER" -h "$POSTGRES_HOST" -p "$POSTGRES_PORT" -d "$POSTGRES_DB"'
```

Comment:
I think it's not needed to run this command through `bash -c`.
It can be as it was earlier.

Reply:
Yes, the `bash -c` is something that I don't like either, but it's needed for the hack in order to return the corrent status code of the `docker exec` command. Otherwise, without this hack, `docker exec` always returns 0 regardless of whether the command in the container failed or not.

Reply:
Are you sure that `docker exec` always return 0?
I tried following code in loop:
```bash
    docker exec ${POSTGRES_CONTAINER} ls /bins
    EXIT_CODE=$?
    echo $EXIT_CODE
```
And it always return 2.

```bash
    docker exec ${POSTGRES_CONTAINER} ls /bin
    EXIT_CODE=$?
    echo $EXIT_CODE
```
returns 0

Reply:
You're actually right. It was my fault, It seems that I've mistaken about this status code.

I've refactored as you suggested, can you test if the script is working okay?

Reply:
Works 😃